eras/byron/ledger/impl/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Revision history for `cardano-ledger-byron`
 
+## 1.3.0.0
```

Comment:
Please also bump the version in `eras/byron/ledger/impl/cardano-ledger-byron.cabal` to match this

Reply:
Thanks, done :+1: